xblock/mixins.py

```diff
@@ -272,7 +272,8 @@ def force_save_fields(self, field_names):
                 fields.remove(field)
                 # if the field was dirty, delete from dirty fields
                 self._reset_dirty_field(field)
-            raise XBlockSaveError(saved_fields, fields)
+            msg = 'Error saving fields {}'.format([key for key in fields])
```

Comment:
Yes; `fields` are the list of `dirty_fields`, so we can log to get a sense which fields are causing the problem.